docs/tutorial.md

```diff
@@ -556,3 +558,29 @@ status:
 
 The status of type `Succeeded = True` shows the pipeline ran successfully, also
 the status of individual Task runs are shown.
+
+
+## Local development
+
+### Known good configuration
+
+Knative (as of version 0.3) is known to work with:
+- [Docker for Desktop](https://www.docker.com/products/docker-desktop): a version that uses Kubernetes 1.11 or higher. At the time of this document, this requires the *edge* version of Docker to be installed
+- The following [prerequisites](https://github.com/knative/build-pipeline/blob/master/DEVELOPMENT.md#requirements)
+- Setting `host.docker.local:5000` as an insecure registry with Docker for Desktop
+- A local Docker registry: this can be run with
+
+`docker run -d -p 5000:5000 --name registry-srv -e REGISTRY_STORAGE_DELETE_ENABLED=true registry:2`
+
+### Images
+- Any hardcoded image locations should be replaced with `host.docker.internal:5000/myregistry/<image name>` equivalents: look for `PipelineResource` definitions which define an `image` specification
```

Comment:
i might reword this as something like "Any `PipelineResource` definitions of `image` type should be updated to use the local registry by setting the url to  `host.docker.internal:5000/myregistry/<image name>` "

this is a total nitpick but I wanted to point it out b/c flexibility of handling different registries with minimal Pipeline changes is a goal :D

Reply:
Yeah you're exactly right, adding, ty